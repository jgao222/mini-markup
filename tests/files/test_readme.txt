<!-- file.txt -->
<tagName attribute="value"> {
    <childTag> {
        Content goes here!
    }
}