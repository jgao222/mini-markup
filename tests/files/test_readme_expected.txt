<!-- file.txt -->
<tagName attribute="value">
    <childTag>
        Content goes here!
    </childTag>
</tagName>